drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -125,11 +128,11 @@ void test_multi_call_time(void)
         gtimer.reset();
 
         gtimer.start();
-        ticker.attach_us(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME_MS * 1000);
+        ticker.attach(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME_US);
         while (!ticker_callback_flag);
-        time_diff = gtimer.read_us();
+        time_diff = gtimer.elapsed_time().count();
 
-        TEST_ASSERT_UINT32_WITHIN(TOLERANCE_US(MULTI_TICKER_TIME_MS * 1000), MULTI_TICKER_TIME_MS * 1000, time_diff);
+        TEST_ASSERT_UINT32_WITHIN(TOLERANCE_US(MULTI_TICKER_TIME_US.count()), MULTI_TICKER_TIME_US.count(), time_diff);
```

Comment:
I copy-and-pasted a `TEST_ASSERT_DURATION_WITHIN` into a bunch of tests that is a bit neater than doing this. See if you can find it. Didn't manage to spot a nice place to put it for a central definition though. Sorry!